### DISCUSSION TITLE
Comment Tag or Sign

### DISCUSSION THREAD

**Discussion by Codeschreibaer** [Acly/krita-ai-diffusion#1155] [Ideas]
Abiltiy of out-commenting text from rest of the prompt would be nice. (Like "//" or "***" in some prog languages.
This could be useful for planning further prompt variancies, or to note which model was used for whoch trial. (When i test for specific qualities like realistic posing or else, i use several models in the same *.kra - testfile. Ability of writing which models was used, i a great help, especially for bigger projects.)

**Comment by parella007** [comment]
I was thinking of something like that, it would be so nice...  I try a lot of combinations, in the same file, i can keep track by making lots of styles and i know, right now we can recover prompt, strength and seed of the images via right click, but maybe, i dont know, having  some metadata embedded in the images, that keeps the prompt, strenght and seed but also: model, sampler, cfg, steps, would be awesome!
